spec/controllers/issues_controller_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe IssuesController, type: :controller do
```

Comment:
You have method [create](https://github.com/Kotany-IF070-WebUI-RoR/NewVasjuky/pull/28/files#diff-7ab71e5c63eb9fdf6fa63cf71ea5ab3bR8), but I don't see spec for it.